pkg/odo/cli/catalog/list/components.go

```diff
@@ -43,7 +43,11 @@ func (o *ListComponentsOptions) Complete(name string, cmd *cobra.Command, args [
 	o.Context = genericclioptions.NewContext(cmd)
 	o.catalogList, err = catalog.ListComponents(o.Client)
 	if err != nil {
-		return err
+		if experimental.IsExperimentalModeEnabled() {
+			log.Warning("Please log in to an OpenShift cluster to list OpenShift/s2i components")
```

Comment:
Rather than pulling in the Go log library, can you use `glog` and log the message with an appropriate log level? LIke we've done elsewhere in odo.

Reply:
New to `glog`, I can switch to use that, and this is a good point. I find odo uses various logging packages (eg glog, log, and some of them directly use fmt) and I don't find any docs mentions odo uses them for different purpose, make the logging packages consistent are good for both development and use cases (this is the only docs for logging that I find: https://github.com/openshift/odo/blob/5eb0ff5e73ad76322724e0cc9ef3beb9f016c72e/docs/dev/logging.adoc). I can bring this up in Slack channel.